packages/document/main-doc/docs/en/community/blog/overview.md

```diff
@@ -4,7 +4,72 @@ sidebar_position: 1
 
 # Overview
 
-Welcome to the Modern.js blog!
+Uhhhh welcome to Modern.js' blog x
 
-You can find the latest news about Modern.js here.
+You can um...find the latest update of Modern.js and our thought processes here. Occasionally we explain the infrastructure behind it.
 
+---
+
+## Announcing Modern.js v2: support Rspack
+
+> Published on 16.03.2023
+
+Haiyaaaaa~~ we are excited to announce the release of Modern.js v2!
+
+Modern.js is a suite of infrastructure tools we built for web development inside Bytedance (we call ourself Web-Infra-Dev internally...). Since we open sourced this project a little more than a year ago, there were dozens of contributors helped us on development; we aggregated more than 2,000 pull requests and Modern.js started to support build tool like Rspack, features like nested routes, Streaming SSR(Server-side Rendering) and there are more to come!
```

Comment:
```suggestion
Modern.js is a suite of infrastructure tools we built for web development inside Bytedance (we call ourself Web Infra). Since we open sourced this project a little more than a year ago, there were dozens of contributors helped us on development; we aggregated more than 2,000 pull requests and Modern.js started to support build tool like Rspack, features like nested routes, Streaming SSR and there are more to come!
```